docs/security/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Introduction
```

Comment:
Title should ideally say the name of the overall section, like "Security and compliance overview". Something useful for SEO. I didn't realize our migration guides index just says introduction :/ That would actually be great to fix either in this PR or another

Ditto maybe making the h1 more useful